packages/javascript/bh-shared-ui/src/hooks/useExploreGraph/useExploreGraph.tsx

```diff
@@ -66,5 +76,6 @@ export const useExploreGraph = (includeProperties?: boolean) => {
                 autoHideDuration: SNACKBAR_DURATION_LONG,
             });
         },
+        enabled,
```

Comment:
it seems that adding this `enabled` field at the end of the properties will override the `enabled` defined within each `queryConfig`, which results in the query trying to execute without a `queryFn` yet.

Perhaps we need to move the enabled field up above the queryConfig, or pass it to `getQueryConfig`?